src/arrayMethodJoin.js

```diff
@@ -6,6 +6,24 @@
 function applyCustomJoin() {
   [].__proto__.join2 = function(separator) {
     // write code here
+    let joined = '';
+    let realSeparator = separator;
```

Comment:
why?

Reply:
This is why

![image](https://user-images.githubusercontent.com/55781950/230038168-1b730960-43fd-4bac-8f23-46a48d3635f1.png)

Reply:
Nevermind the thing I deleted, made a typo... my entire reason for using a new variable for the separator ended up being a typo, I'm sometimes amazed at the dumb mistakes I make